source/sharding.rst

```diff
@@ -22,15 +25,71 @@ A MongoDB Sharding involves the following components:
    cluster; still, this limited sharding support allows using ``mongos`` as an
    entry point instead of provisioning a load-balancer per replica set node.
 
+Turning sharding on and off
+---------------------------
+
 Sharding is controlled by the ``sharding`` section of the ``deploy/cr.yaml``
 configuration file and is turned on by default.
 
 To enable sharding, set the ``sharding.enabled`` key ``true`` (this will turn
-existing MongoDB replica set nodes into sharded ones).
+existing MongoDB replica set nodes into sharded ones). To disable sharding, set
+the ``sharding.enabled`` key ``false``.
 
 When the sharding is turned on, the Operator runs replica sets with config
 servers and mongos instances. Their numbers are controlled by 
 ``configsvrReplSet.size`` and ``mongos.size`` keys respectively.
 
 .. note:: Config servers for now can properly work only with WiredTiger engine,
    and sharded MongoDB nodes can use either WiredTiger or InMemory one.
+
+Checking connectivity to sharded and non-sharded cluster
+--------------------------------------------------------
+
+With sharding turned on, you have ``mongos`` service as an entry point to access
+your database. If you do not use sharding, you have to access ``mongod``
+processes of your replica set.
+
+1. Run percona-client and connect its console output to your terminal (running
+   it may require some time to deploy the correspondent Pod): 
```

Comment:
```suggestion
1. Run percona-client and connect its console output to your terminal (running
   it may require some time to deploy the corresponding Pod): 
```